pulsar/message.go

```diff
@@ -166,6 +166,12 @@ type MessageID interface {
 	String() string
 }
 
+// TopicMessageID defined the correspondence between topic and MessageID.
+type TopicMessageID struct {
```

Comment:
Is it better to make it as an interface? And I think it should also implement the `MessageID` interface.

Reply:
Good suggestion, keep the external API consistent.